### DISCUSSION TITLE
Reporting references / citations

### DISCUSSION THREAD

**Discussion by dtoher** [easystats/report#302] [General]
For example, in the report_effectsize function, the output cites references without providing the actual references anywhere. For example, Field (2013) is mentioned, but the actual full reference should, at the minimum, be in the help file.

Ideally, adding an option to allow the user to save the citation in a bibtex format to allow for more comprehensive referencing using different styles (especially by using different cls files) would be the way to go - I would suggest having this option set as FALSE as default and that it wouldn't be printed to screen unless specifically requested.

**Comment by bwiernik** [comment]
That's an oversight. This is the reference:

```
@book{field2013discovering,
  title={Discovering statistics using IBM SPSS statistics},
  author={Field, Andy},
  year={2013},
  publisher={sage}
}
```
